### ISSUE TITLE
Weblogo 3.7 fails to parse --title and --fineprint strings containing spaces

### ISSUE THREAD

**Issue by bfristensky** [WebLogo/weblogo#102]
In Weblogo 3.4, strings accompanying --title and --fineprint options could be quoted to allow blank spaces:

$birch/pkg/weblogo-3.4/weblogo -f chitIII.pro.tcoffee.fsa --title 'two words' > test.eps

ran correctly, producing an EPS file. The specified title correctly appears in the output.

Weblogo 3.7 appears to parse quoted string arguments as separate tokens:

weblogo -f chitIII.pro.tcoffee.fsa --title 'two words' > test.eps
Usage: weblogo [options]  < sequence_data.fa > sequence_logo.eps

weblogo: error: Unparsable arguments: ['words'] 

(In the latter example, the shell is calling weblogo 3.7.)

I have tried double quotes, and escaping the blank character with a backslash, which give the same result. This problem is also seen with the --fineprint option.

Apparently, something in the command line parser changed after v3.4.

**Comment by gecrooks** [comment]
I can't reproduce this issue. Works fine with 3.7 on my machine. What operating system, shell, and python version are you using?

**Comment by bfristensky** [comment]
This is Red Hat Enterprise Linux 7. Python is v3.6.8, and I'm using the bash shell.

**Comment by gecrooks** [comment]
You say " In the latter example, the shell is calling weblogo 3.7". The initial tokenization of a command line takes place at the shell level, not at the python level. Are you sure your shell isn't doing something funky before weblogo gets called?

**Comment by bfristensky** [comment]
> You say " In the latter example, the shell is calling weblogo 3.7". The initial tokenization of a command line takes place at the shell level, not at the python level. Are you sure your shell isn't doing something funky before weblogo gets called?

To be specific, Weblogo3.7 is in a bin directory listed in the $PATH, whereas I launched Weblogo3.4 by typing the full path to the weblogo script. Otherwise, the commands are identical. Recognizing that the shell does the first round of parsing the command line, 3.4 is able to take --title 'some words with blanks'  and create a logo, while 3.7 gets the same --title 'some words with blanks' and gives the error message.

Today I also tried this on a Ubuntu 18.04 LTS system running Python3.6.9, with the same result.
